src/private_types.h

```diff
@@ -521,6 +523,11 @@ struct ecs_stage_t {
     /* Caches for rule creation */
     ecs_vec_t variables;
     ecs_vec_t operations;
+
+#ifdef FLECS_PIPELINE
```

Comment:
Nitpick, but since there is only one pipeline being ran at any given point in time, can this value be moved to `ecs_world_t`?

Reply:
Yep, makes sense.

It did spark a thought that if in the future flecs gets the ability to schedule different systems on different workers then something like this may return, but that is a future project 😄.